### DISCUSSION TITLE
What would be needed to use Kagi.com to search (and also fastGPT from Kagi)?

### DISCUSSION THREAD

**Discussion by ghost** [armindarvish/consult-omni#17] [Q&A]
Hi, what would be needed to add support to search using Kagi search engine?
And also to add support to fastGPT from Kagi?

**Comment by armindarvish** [comment]
@oscarvarto nothing is needed, it is already supported through the gptel source!

If for some reason you need a standalone kagi source, then watch the video here: https://www.youtube.com/watch?v=wNH2E7iT__c&t=9245s

and follow the example of the provided [chatgpt source](https://github.com/armindarvish/consult-omni/blob/main/sources/consult-omni-chatgpt.el) for simple API calls!

Closing this now
